packages/cli/generators/model/templates/model-extended.ts.ejs

```diff
@@ -0,0 +1,20 @@
+import {model, property} from '@loopback/repository';
+import {<%= modelBaseClass %>} from './';
+
+@model()
+export class <%= className %> extends <%= modelBaseClass %> {
+<% Object.entries(properties).forEach(([key, val]) => { -%>
+  @property({
+  <%_ Object.entries(val).forEach(([propKey, propVal]) => { -%>
+    <%_ if (!['tsType'].includes(propKey)) { -%>
```

Comment:
I am not very happy that we have this complex logic duplicated in two templates. Can we us a simple `if` in the original template to decide what to import?

```ejs
<% if(isModelBaseBuiltin) -%>
import {<%= modelBaseClass %>, model, property} from '@loopback/repository';
<% } else { -%>
import {model, property} from '@loopback/repository';
import {<%= modelBaseClass %>} from '.';
<% } -%>
```

(Note that `require('./')` can be shortened to `require('.')` in recent Node.js versions.)

Reply:
Ok. The idea was to separate them at this stage in case in the future both receive different complex logic. But at this point it makes sense to place the if as you indicated 👍